source/authentication/pam.rst

```diff
@@ -0,0 +1,76 @@
+.. _authentication-tutorial-pam-authnz:
+
+PAM Authentication using mod Authnz
+=======================================
+
+These are the steps you'll need to take to enable PAM authentication in
+OnDemand using the ``mod_authnz_external`` httpd plugin.  These steps are
+community provided, so if you have issues with them, let us know!
+
+
+#. Enable epel if required
+
+    .. code-block:: sh
+
+       yum install epel-release
+
+#. Install pwauth
+
+    .. code-block:: sh
+
+       yum install pwauth
+
+
+#. Download the mod_authnz_external rpm
+
+    .. code-block:: sh
+
+       wget http://download.fedoraproject.org/pub/epel/7/x86_64/Packages/m/mod_authnz_external-3.3.1-7.el7.x86_64.rpm
+
+
+#. Unpack the files from the rpm.
+
+    .. code-block:: sh
+
+       rpm2cpio mod_authnz_external-3.3.1-7.el7.x86_64.rpm | cpio -idmv
+
+
```

Comment:
For RHEL/CentOS 7 there should be mention of `yum install mod_authnz_external` that is done instead of the two commands of `wget` and `rpm2cpio`.

Reply:
Users have to dl and unpack because the rpm installation uses / instead of /opt/rh/httpd24/root.  Right? I thought that's why these folks did this.

Reply:
Actually kind of.  Part of the issue is they wanted RHEL7 RPM on RHEL6 so only way to do that is download and extract. Also now that I think about it the extract would be necessary on RHEL7 too to avoid installing the system Apache which is almost certainly a dependency. So installing the RPM would be a problem.